pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -76,31 +63,31 @@ func getDefaultInstrumentation() (*v1alpha1.Instrumentation, error) {
 			Java: v1alpha1.Java{
 				Image: javaInstrumentationImage,
 				Env: []corev1.EnvVar{
-					{Name: otelSampleEnabledKey, Value: otelSampleEnabledDefaultValue}, //TODO: remove in favor of new name once safe
-					{Name: otelAppSignalsEnabledKey, Value: otelAppSignalsEnabledDefaultValue},
-					{Name: otelTracesSamplerArgKey, Value: otelTracesSamplerArgDefaultValue},
-					{Name: otelTracesSamplerKey, Value: otelTracesSamplerDefaultValue},
-					{Name: otelExporterOtlpProtocolKey, Value: otelExporterOtlpProtocolValue},
-					{Name: otelExporterTracesEndpointKey, Value: otelExporterTracesEndpointDefaultValue},
-					{Name: otelExporterSmpEndpointKey, Value: otelExporterSmpEndpointDefaultValue}, //TODO: remove in favor of new name once safe
-					{Name: otelExporterAppSignalsEndpointKey, Value: otelExporterAppSignalsEndpointDefaultValue},
-					{Name: otelExporterMetricKey, Value: otelExporterMetricDefaultValue},
-					{Name: otelExporterLogsKey, Value: otelExporterLogsDefaultValue},
+					{Name: "OTEL_SMP_ENABLED", Value: "true"}, //TODO: remove in favor of new name once safe
+					{Name: "OTEL_AWS_APP_SIGNALS_ENABLED", Value: "true"},
+					{Name: "OTEL_TRACES_SAMPLER_ARG", Value: "endpoint=http://cloudwatch-agent.amazon-cloudwatch:2000"},
```

Comment:
I still see this hardcoded as `http` for both java and python. Is that intentional?

Reply:
yes, our aws proxy still uses http. our tls setting only affects the agent otlp receiver which means we should only be changing the otlp exporter path for the adot sdk

Reply:
Is that by design? Or should we fix that in a follow up later?

Reply:
Since this also affects the traces functionality, probably this needs to be follow up. But I also don't think encryption for sampling args is as critical as encryption of metrics within a cluster.